src/scrollSpy/scrollSpyAreaDirective.js

```diff
@@ -0,0 +1,36 @@
+const _window = new WeakMap();
```

Comment:
I've read that we can create private-as fields (for example, in this article https://www.sitepoint.com/writing-angularjs-apps-using-es6/#defining-controllers)